### ISSUE TITLE
UI: Find all untranslatable strings, fix that.

### ISSUE THREAD

**Issue by Shelob9** [Shelob9/ingot#47]

**Comment by Shelob9** [comment]
Should be gone now. If anymore are found, they should be annihilated and past SHP awards will be removed.
